README.md

```diff
@@ -1,69 +1,89 @@
 
-1.1. Scope
+[@brightsign/filesysteminfile](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/404622545/filesysteminfile)
 
-Documentation on the usage for Brightsign Javascript APIs:
+[@brightsign/usbfilesystem](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/404622824/usbfilesystem)
 
-@brightsign/filesysteminfile
 
-@brightsign/usbfilesystem
+## Example BrightAuthor:connected Presentation
 
-
-2. Instructions
-
-2.1. Updating USB Filesystem
+The example presentation and HTML file encompasses one approach for how you may send messages to HTML. The .html file is meant to highlight how to implement and utilize the filesystem and usb APIs. 
 
 UDP commands are utilized throughout the sample presentation provided. The correct destination IP and port should be set in order for the Javascript app to receive the UDP commands sent from BrightAuthor:connected to the app. 
 
 In the Presentation view under  “Presentation Settings” -> “Interactive” -> “Networking” -> “Specific IP Address” can be 127.0.0.1 and UDP Destination Port should be 5000. 
 
-2.2. BrightAuthor Integration
 
-In BrightAuthor:connected, add the html / JavaScript file by adding an HTML Widget onto the interactive plane. From here, sending the UDPs mentioned below can be sent from anywhere in the presentation or from other Javascript to the destination IP and destination port. 
+## BrightAuthor:connected  Integration
+
+In BrightAuthor:connected, add the html / JavaScript file by adding an HTML Widget onto the interactive plane. From here, sending the UDPs mentioned below can be sent from anywhere in the presentation or from other Javascript processes to the destination IP and destination port. 
 
-Command Parameter Structure: usb!command!command parameter (optional)
+Command Parameter Structure
+```
+usb!!<command>!!<command parameter (optional)>
+```
 
-2.3. Managing Filesystem
+### Managing Filesystem
 
 There are a couple commands which should be sent in order to allow for the app to understand how it should create the filesystem and the contents that are copied. 
+___
 
-Command: usb!startfilesystem
+_Command_: 
+```
+usb!!startfilesystem
+```
 
-Description: Sends a command to notify the app to acknowledge when to start listening to the addasset UDP commands below. 
+_Description_: Sends a command to notify the app to acknowledge when to start listening to the addasset UDP commands below. 
+___
 
-Command: usb!addasset!<filename or directory>
-  
-Description: Sends a command to notify the app what is the name of a file or directory that should be copied to the usb drive. This command can be sent multiple times.
+_Command_: 
+```
+usb!!addasset!!<filename or directory>
+```
 
-Note: The current expectation of this example is that the source dependencies live on the root of the SD card.
+_Description_: Sends a command to notify the app what is the name of a file or directory that should be copied to the usb drive. This command can be sent multiple times.
```

Comment:
Is the file or directory also to the root of usb drive?

Reply:
The way the logic is written, it looks like the the file can be added into nested levels. This example is not comprehensive and to be used as one example as how to possibly use these objects. 

Looking back over the script there could be many improvements.